src/containers/lims/LIMSTable/index.tsx

```diff
@@ -152,8 +152,7 @@ function LIMSTable({ defaultQueryParam, sideBar = false }: Props) {
 
   const handleFilterApplied = useCallback(
     (filteredQueryParam: Record<string, string[] | number[]>) => {
-      setApiQueryParameter((prev) => ({
-        ...prev,
+      setApiQueryParameter(() => ({
```

Comment:
Right. Removing `prev` state ok. 

In that case, could you also add the default sort `ordering` column and `rowsPerPage` setting. Like so:

```
      setApiQueryParameter(() => ({
        rowsPerPage: paginationProps.currentNumberOfRows,
        ordering: '-subject_id',
        ...filteredQueryParam,
      }));

```

That should perfect the UX flow.

Reply:
sure, didn't realize there are default settings in front. Thank you for that 🫡